tests/loops/test_loops.py

```diff
@@ -731,7 +733,130 @@ def test_fit_loop_reset(tmpdir):
     assert epoch_loop.restarting
     assert epoch_loop.batch_progress.total.ready == 4
     assert epoch_loop.batch_progress.total.completed == 3  # the checkpoint was saved on train_batch_end
-    assert epoch_loop.batch_progress.current.ready == 0
-    assert epoch_loop.batch_progress.current.completed == 0
+    assert epoch_loop.batch_progress.current.ready == 4
+    assert epoch_loop.batch_progress.current.completed == 4
```

Comment:
This value change doesnt look right

We are loading a ckpt from the "end of the epoch" and `.reset()` has been called.


Why would they not be 0?

Reply:
Yes, added a mechanism to enable this.